setup.py

```diff
@@ -9,8 +9,16 @@
 here = os.path.abspath(os.path.dirname(__file__))
 about = {}
 
-with open(os.path.join(here, "requirements.txt"), "r") as f:
-    requirements = f.read().strip().split("\n")
+requirements = ['SQLAlchemy==1.3.23',
```

Comment:
can we put it in a file?  Similar to requirements.txt? Just another name

Reply:
@eugenemiretsky